vertx-grpc-server/src/main/java/io/vertx/grpc/server/impl/GrpcServerRequestImpl.java

```diff
@@ -213,6 +243,61 @@ private boolean notGrpcWebText() {
     return grpcWebTextBuffer == null;
   }
 
+  public boolean isTranscodable() {
+    return (httpRequest.version() == HttpVersion.HTTP_1_0 || httpRequest.version() == HttpVersion.HTTP_1_1) && GrpcProtocol.HTTP_1.mediaType()
+      .equals(httpRequest.getHeader(CONTENT_TYPE));
+  }
+
+  private Buffer mutateMessage(Buffer message, List<HttpVariableBinding> bindings) {
```

Comment:
we should move this code to the transcoding module so it can be tested there independently  of gRPC interactions.